Modules/Core/ImageFunction/include/itkCentralDifferenceImageFunction.h

```diff
@@ -217,37 +217,43 @@ class ITK_TEMPLATE_EXPORT CentralDifferenceImageFunction : public ImageFunction<
     using Type = T;
   };
 
-  /** Specialized versions of EvaluateAtIndex() method to handle scalar or vector pixel types.*/
+  /** Specialized version of EvaluateAtIndex() method to handle scalar pixel types.*/
   template <typename Type>
   inline void
   EvaluateAtIndexSpecialized(const IndexType & index,
                              OutputType &      orientedDerivative,
                              OutputTypeSpecializationStructType<OutputType>) const;
+
+  /** Specialized version of EvaluateAtIndex() method to handle vector pixel types.*/
   template <typename Type>
   inline void
   EvaluateAtIndexSpecialized(const IndexType & index,
                              OutputType &      derivative,
                              OutputTypeSpecializationStructType<Type>) const;
 
-  /** Specialized versions of EvaluateAtContinuousIndex() method to handle scalar or vector pixel types.*/
+  /** Specialized version of EvaluateAtContinuousIndex() method to handle scalar pixel types.*/
   template <typename Type>
   inline void
   EvaluateAtContinuousIndexSpecialized(const ContinuousIndexType & cindex,
                                        OutputType &                orientedDerivative,
                                        OutputTypeSpecializationStructType<OutputType>) const;
+
+  /** Specialized version of EvaluateAtContinuousIndex() method to handle vector pixel types.*/
   template <typename Type>
   inline void
   EvaluateAtContinuousIndexSpecialized(const ContinuousIndexType & cindex,
                                        OutputType &                derivative,
                                        OutputTypeSpecializationStructType<Type>) const;
 
-  /** Specialized versions of Evaluate() method to handle scalar or vector pixel types.*/
+  /** Specialized version of Evaluate() method to handle scalar pixel types.*/
```

Comment:
Is the removal of `or vector` intended?

Reply:
Yes Matt. It looks like there are two versions for each of these methods, one for scalar types and the other one for vector types; at least that was what was being said in the implementation file. Please check.